packages/mux-audio/src/index.ts

```diff
@@ -79,7 +79,15 @@ class MuxAudioElement extends CustomAudioElement<HTMLAudioElement> implements Pa
     return playerSoftwareVersion;
   }
 
+  /**
+   * @deprecated please use ._hls instead
+   */
   get hls() {
+    console.warn('<mux-player>.hls is deprecated, please use ._hls instead');
+    return this._hls;
+  }
+
+  get _hls() {
```

Comment:
Since we're using this property, it'll mean that the default example will log this warning as well. Ideally, there shouldn't be any such warnings by default.